nyxx/lib/src/core/message/Message.dart

```diff
@@ -174,6 +174,12 @@ abstract class Message extends SnowflakeEntity implements Disposable {
   Future<void> unpinMessage() =>
       client._httpEndpoints.unpinMessage(this.channel.id, this.id);
 
+  // TODO Return actual thread channel
```

Comment:
I think it should be left as is and crate additional method like `createAndGetThread` which would create then fetch channel aditionally

Reply:
Done!